src/main/scala/codechicken/multipart/api/part/TMultiPart.scala

```diff
@@ -110,14 +110,18 @@ abstract class TMultiPart {
      */
     def occlusionTest(npart: TMultiPart): Boolean = true
 
-    def getOutlineShape: VoxelShape = VoxelShapes.empty()
+    def getOutlineShape(context:ISelectionContext): VoxelShape = VoxelShapes.empty()
 
     //TODO pass through ISelectionContext
-    def getCollisionShape: VoxelShape = getOutlineShape
+    def getCollisionShape(context: ISelectionContext): VoxelShape = getOutlineShape(context)
 
-    def getRenderOcclusionShape: VoxelShape = getOutlineShape
+    def getRenderOcclusionShape: VoxelShape = getOutlineShape(ISelectionContext.empty)
```

Comment:
This should be renamed to 'getOcclusionShape', however, might conflict with other things.